python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -975,9 +988,74 @@ def do_evaluate(
             child.evaluate(df, context=context, mapping=mapping)
             for child in self.children
         ]
-        if self.name == pl_expr.TemporalFunction.Year:
+        if self.name in TemporalFunction._temporal_extraction_function_map:
             (column,) = columns
-            return Column(plc.datetime.extract_year(column.obj))
+            if self.name == pl_expr.TemporalFunction.Millisecond:
+                return Column(
+                    plc.datetime.extract_datetime_component(column.obj, "millisecond")
+                )
+            if self.name == pl_expr.TemporalFunction.Microsecond:
+                milis = plc.datetime.extract_datetime_component(
```

Comment:
Typo: Use “millis” instead of “milis.”
```suggestion
                millis = plc.datetime.extract_datetime_component(
```